### ISSUE TITLE
Some Major Issue on low Cheap Budget phone Please Read Comment

### ISSUE THREAD

**Issue by zyroxom** [naz664/crDroid_patches#37]
First of all, my prayers🤲, love ❤and respect 🦾for those who are working behind this project 🤗And F stock ROM🤮. can't use mid range low or cheap budget phones without ads, sponsors, promotion and bunch of bloatwares F-ing Up phone performance and battery, notification after notification, not optimised at all.

**Major Issue on low Cheap Budget phone**
1. Wifi, hostspot and Bluetooth not working
2. loud speaker not working in call
3. Volume/Sounds Very Low Everywhere
this 3 major issue must need to fix

**Not So Big Deal/Small/Minor Issues**
1. double tap to wake up not working
2. Front Micro LED not Work(When Charging/Notification And Others)
3. Quick Torch Not Working
4. restarts takes 2:00-2:30 Minutes

 **One Request**
1. Need a Vanila De-googled Version (With MicroG Services+Disable Signature Verification)

My Cheap  Phone Model:
INFINIX HOT 11 PLAY X688B
4/64
HELIO G35
HERE IS A LINK FOR STOCK ROM: www.firmwarefile.com/infinix-hot-play-x688b

**Comment by naz664** [comment]
Using custom vendor/kernel ? 
If yes reflash stock and check
if no, then test if it works on trebleDroid aosp.

**Comment by zyroxom** [comment]
Hello Assalamu Alikum Brother Nazzim😊Thanks For Your Reply, I Hope You Are Well , I'm Not Using Any Custom Vendor/Kernel Brother. 
I Used Pixel Experience Plus {Link: https://github.com/ponces/treble_build_pe/releases} it was almost perfect, but it has some same issue. BTW Wi-Fi and bluetooth is working perfectly with Pixel Experience Plus no issue at all.

The Main Issue with Pixel Experience Plus:
1. loud speaker not working in call
2. Volume/Sounds Very Low Everywhere
3. Front Micro LED not Work(When Charging/Notification And Others) BTW it's not a big issue.
4. after 3-4 days it become very slow and heavy on system sometime opening apps takes 4 to 5 second.

I tried all of your GSI Roms Crdroid GSI, SparkOS GSI, ArrowOS GSI
they all have same problem.

1. Wifi, hostspot and Bluetooth not working
2. loud speaker not working in call
3. Volume/Sounds Very Low Everywhere

SparkOS GSI Has A Another Problem 
When I Open Magsik It Says, "abnormal state - "su" binary not from magisk has been detected"

With ArrowOS GSI
Random Soft Reboots

brother I have noticed 1 interesting thing
Android phones that have released With android A8/A9/A10/A11 OS. majority of them facing this issues. But Those phones that have released With android 12/12L/12.1/13/14 they are not facing any type of major issues.

I hope brother you try your best To fixed it, waiting for your reply and next release. 😊Love From BD❤

**Comment by zyroxom** [comment]
I think I am requesting too much please don't mind. I am trying to help you so you can fix and grow the community. I am just a normal user I don't understand any technical stuff TBH. I only know how to root your phone, how to install custom ROM, how to block ads on your phones thats it.

with your all GSI just fix two things
1. fix wifi and bluetooth connectivity
2. fix loud speaker in call
thats it.

With SparkOS GSI
When I Open Magsik It Says, "abnormal state - "su" binary not from magisk has been detected"

With ArrowOS GSI
Random Soft Reboots
thats it

**Comment by zyroxom** [comment]
Hello Assalamu Alikum Brother Nazzim😊Thanks For Your Reply, I Hope You Are Well , I'm Not Using Any Custom Vendor/Kernel Brother. 
I Used Pixel Experience Plus and ArrowOS GSI (Your Build) {Link: https://github.com/ponces/treble_build_pe/releases} it was almost perfect, but it has some issue. BTW Wi-Fi and bluetooth is working perfectly with Pixel Experience Plus and ArrowOS GSI (Your Build).

The Main Issue with Pixel Experience Plus and ArrowOS GSI (Your Build):
1. loud speaker not working in call
2. Volume/Sounds Very Low Everywhere
3. Front Micro LED not Work(When Charging/Notification And Others) BTW it's not a big issue.
4. after 3-4 days it become very slow and heavy on system sometime opening apps takes 4 to 5 second.

I tried all of your GSI Roms Crdroid GSI, SparkOS GSI, 
these two also have same problem 

1. Wifi, hostspot and Bluetooth not working {except Pixel Experience Plus and ArrowOS GSI Working Fine With These two GSI}
2. loud speaker not working in call 
3. Volume/Sounds Very Low Everywhere

SparkOS GSI Has A Another Problem 
When I Open Magsik It Says, "abnormal state - "su" binary not from magisk has been detected"

With ArrowOS GSI
Random Soft Reboots

brother I have noticed 1 interesting thing
Android phones that have released With android A8/A9/A10/A11 OS. majority of them facing this issues. But Those phones that have released With android 12/12L/12.1/13/14 they are not facing any type of major issues.

in summary:
-----------
1. With Pixel Experience Plus+ArrowOS GSI (Your Build)
Wifi+bluetooth Working

2. With Crdroid GSI SparkOS GSI
Wifi+bluetooth not Working

I hope brother you try your best To fixed it, waiting for your reply and next release. 😊Love From BD❤

**Comment by Evvyinnit** [comment]
@zyroxom hey we both have the same device. Did you found any good GSI that working without performance issues? Thanks!
